client/strings/en-us.json

```diff
@@ -48,6 +48,7 @@
   "ButtonNextChapter": "Next Chapter",
   "ButtonNextItemInQueue": "Next Item in Queue",
   "ButtonOk": "Ok",
+  "ButtonOpenCurrentChapter": "Open Current Chapter",
```

Comment:
We only want to add the string to the English file, not all of the other files unless you know the translation (Weblate will handle the other languages)

Reply:
How do i undo all the changes? do i just undo locally and commit again?

Reply:
If you did all of the changes in one commit you can do `git revert [SHA]`, or checkout the files from a specific commit and then do another commit, or undo it manually.

Thanks for doing that, it makes it easier to review the changes. :)